homeassistant/components/switch/flux.py

```diff
@@ -64,7 +73,7 @@
         vol.Any(MODE_XY, MODE_MIRED, MODE_RGB),
     vol.Optional(CONF_INTERVAL, default=30): cv.positive_int,
     vol.Optional(ATTR_TRANSITION, default=30): VALID_TRANSITION
```

Comment:
We shouldn't use the constant `ATTR_TRANSITION`. All config keys should start with `CONF_`.